docs/getting_started/index.rst

```diff
@@ -1,115 +1,164 @@
 
-Before you begin, you'll want to run the ``security-setup`` script in the
-root directory. This will create and set passwords, authentication, and
-certificates. For more information, see the :doc:`security-setup
-<../security/security_setup>` documentation.
+You can set up your DNS records with Terraform: `dns.rst`_
+
+Provisioning Cloud Hosts
+>>>>>>>>>>>>>>>>>>>>>>>>
+
+Here are some guides specific to each of the platforms that Mantl supports:
+
+- `openstack.rst`_
+- `gce.rst`_
+- `aws.rst`_
+- `digitalocean.rst`_
+- `vsphere.rst`_
+- `softlayer.rst`_
 
 Deploying software via Ansible
-------------------------------
+>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
 
 .. note:: Ansible requres a Python 2 binary. If yours is not at /usr/bin/python,
           please view the `Ansible FAQ <http://docs.ansible.com/faq.html>`_. You
           can add an extra variable to the following commands, e.g.
           ``ansible -e ansible_python_interpreter=/path/to/python2``.
 
-In the following examples, we're going to assume you deployed hosts using
-Terraform. This project ships with a dynamic inventory file to read Terraform
-``.tfstate`` files. If you are running ansible from the root directory of the
-project, this inventory file will be used by default. If not, or to use a custom
-inventory file, you can use the ``-i`` argument of ``ansible`` or
-``ansible-playbook`` to specify the inventory file path. For example:
+The following steps assume that you have provisioned your cloud host by taking
+the steps listed in one of the guides listed above. We're going to assume you
+deployed hosts using Terraform.
+
+This project ships with a dynamic inventory file to read Terraform ``.tfstate``
+files, `terraform.py`_.  If you are not running Ansible from the root directory,
+or would like to use a custom inventory file, you can use the ``-i`` argument
+of ``ansible`` or ``ansible-playbook`` to specify the inventory file path.
 
 .. code-block:: shell
 
-   ansible-playbook -i path/to/inventory -e @security.yml mantl.yml
+       ansible-playbook -i path/to/inventory -e @security.yml mantl.yml
 
-First, ping the servers to ensure they are reachable via ssh:
+Steps to deploying via Ansible:
+>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+
+Step 1: Add password to the ssh-agent
+-------------------------------------
+
+For the next steps, you may want to `add your password to the ssh-agent
+<https://wiki.archlinux.org/index.php?title=SSH_keys&redirect=no#SSH_agents>`_
+avoid re-entering your password.
 
 .. code-block:: shell
 
-  ansible all -m ping
+        eval $(ssh-agent) && ssh-add ~/.ssh/id_rsa
 
-If any servers fail to connect, check your connection by adding ``-vvvv``
-for verbose SSH debugging and try again to view the errors in more detail.
+Step 2: Ping the servers to ensure they are reachable via ssh
+-------------------------------------------------------------
+
+.. code-block:: shell
+
+        ansible all -i plugins/inventory/terraform.py -m ping
```

Comment:
we can remove `-i plugins/inventory/terraform.py`, it's the default inventory plugin (check `ansible.cfg`)